src/rules/comment-empty-line-before/README.md

```diff
@@ -116,3 +116,32 @@ a {
   color: pink;
 }
 ```
+
+### `ignore: ["between-comments"]`
+
+Don't require an empty line between comments.
+
+For example, with `"always"`:
+
+The following patterns are considered warnings:
+
+```css
+a {
+  background: pink;
+  /* comment */
+  /* comment */
+  color: #eee;
+}
+```
+
+The following patterns are *not* considered warnings:
+
+```css
+a {
+  background: pink;
+
+  /* comment */
+  /* comment */
+  color: #eee;
+}
+```
```

Comment:
I actually find the first example here, the warning, kind of confusing. I think it would be a good idea to just show patterns that are *not* considered warnings because of this ignore option. Both the one that you have, and also one with a newline between comments (because that's skill ok).